plugins/woocommerce/composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "woocommerce/woocommerce",
-	"description": "An eCommerce toolkit that helps you sell anything. Beautifully.",
+	"description": "An eCommerce toolkit that helps you sell anything. Beautifully...",
```

Comment:
Is this accidental?

Reply:
Triggers build - I'm trying to add a pre-build version of the reporter, as Adi mentioned earlier. Will cleanup before merge.

Reply:
Cleaned up